test/parallel/test-crypto-engine.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const common = require('../common');
+
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+
+const assert = require('assert');
+const crypto = require('crypto');
+
+assert.throws(function() {
+  crypto.setEngine(true)
+}, /"id" argument should be a string/);
+
+assert.throws(function() {
+  crypto.setEngine('/path/to/engine', 'notANumber')
```

Comment:
Does the linter pass with these (`make lint`)? We do use semicolons to end statements :smile: